nixos/modules/services/databases/openldap.nix

```diff
@@ -55,7 +55,7 @@ in
 
       configDir = mkOption {
         type = types.path;
```

Comment:
thanks TIL about `types.nullOr` I'll followup on this when I get time later next week.